### ISSUE TITLE
[Localization issue] page 21 "Customer Card" - NA vs. W1/NL

### ISSUE THREAD

**Issue by Duizy1971** [microsoft/ALAppExtensions#26029]
### Describe the request

The W1 and at least the NL release contain a new BC23 action "Report Customer - Labels" with report::"Customer - Labels" on page::"Customer Card". Alas it  does not exist in the NA localization. I can understand that because they already have an action "Customer Labels" with report :"Customer Labels NA".
The thing is that we base our addon on Base-W1 release and can always use it with a Base-NL. 
Now we are getting a SaaS customer which is on NL and NA and we cannot get the Appsource validated againt the Base-US.
This is often an issue with the heavily customized NA localization. Previously we had an issue with report 113 "Customer/Item Sales" which did not exist at all. Could you please fix this so that our page extension does not fail. I cannot make breaking changes and are already using a reference to action "Report Customer - Labels" in our Appsource app.

            action("Report Customer - Labels")
            {
                ApplicationArea = Basic, Suite;
                Caption = 'Customer - Labels';
                Image = "Report";
                //The property 'PromotedCategory' can only be set if the property 'Promoted' is set to 'true'
                //PromotedCategory = Category9;
                ToolTip = 'View mailing labels with the customers'' names and addresses.';

                trigger OnAction()
                begin
                    RunReport(REPORT::"Customer - Labels", Rec."No.");
                end;
            }

            action("Customer Labels")
            {
                ApplicationArea = Basic, Suite;
                Caption = 'Customer Labels';
                Image = "Report";
                //The property 'PromotedCategory' can only be set if the property 'Promoted' is set to 'true'
                //PromotedCategory = "Report";
                ToolTip = 'Create mailing labels with the customers'' names and addresses. The report can be used to send sales letters, for example.';

                trigger OnAction()
                begin
                    RunReport(REPORT::"Customer Labels NA", Rec."No.");
                end;
            }

### Additional context

Cannot use the same app on Base-NA and cannot create a breking change to make that happen.

**Comment by mynjj** [comment]
@Duizy1971 we can add the action in NA with `Visible = false` to avoid confusing people that will now see two "Customer - Labels" in the NA localization. Would that be ok? Or in what way are you relying on this action?

Proper deprecation of the NA report would be out the scope of this extensibility request.

**Comment by Duizy1971** [comment]
No Breaking Changes. Just always add everything from W1 to NA and do not skip controls/code etc. that can be extended against. 
As you already suggested: Set Visible=false of the W1 actions in the NA localized Base will suffice. 
AppSource can only contain 1 app which should be able to build to any of the still existing localized Microsoft Base apps.
In our azure devops we have 7 extra pipelines to make sure that Base nl, be, de, us etc. will still build when our developers only develop and test against a Base w1.

**Comment by mynjj** [comment]
You are right. You can wait this specific one in 24.1, the more general request is a harder problem that's related to the delocalization efforts being worked on.

Thanks for reporting the issue.

(if you are just relying on the action for addafter, addbefore, it would probably be better to develop using addlast to avoid these issues, it of course depends on the scenario)

**Comment by JesperSchulz** [comment]
Availability update: We will publish a fix for this issue in the next update for release 24.

Build ID to track: 18235.
